base/src/main/java/io/spine/validate/FieldValidatingOption.java

```diff
@@ -46,6 +50,29 @@ boolean shouldValidate(FieldValue<F> value) {
         return valueFrom(value).isPresent();
     }
 
+    @Override
+    public Optional<T> valueFrom(FieldValue<F> bearer) {
```

Comment:
Please rename the parameter to `value` (similarly to one passed to `optionValue()`).